### PR TITLE
Factory reset function (manual EEPROM format)

### PR DESCRIPTION
**Pull request by kilrah**
To be discussed for inclusion.
Did it mostly for myself for testing, but we did have some people request a way to reset their radio to factory defaults in the past.

Basically, the question is "do we trust users to understand they're erasing everything"...

### CODE REVIEW

File: radio/src/eeprom_rlc.cpp [hunk 1]
```diff
@@ -1016,6 +1016,21 @@ void eeLoadModel(uint8_t id)
   }
 }
 
+void eeErase(bool warn)
```
Comment (bsongis) [inline radio/src/eeprom_rlc.cpp]: Not too much falsh added on stock 9X?

File: radio/src/translations/en.h.txt [hunk 2]
```diff
@@ -925,6 +925,8 @@
 #define TR_INPUTS              "Inputs"
 #define TR_OUTPUTS             "Outputs"
 #define TR_EEBACKUP            "\004[ENTER Long] to backup the EEPROM"
+#define TR_FACTORYRESET        "\012[MENU Long]: Factory reset"
```
Comment (bsongis) [inline radio/src/translations/en.h.txt]: I would see a contextual menu + a confirmation, no?

**Comment by bsongis** [comment]
Yes I think it's a needed feature, at least until the EEPROM format v217 is in stone

File: radio/src/translations/en.h.txt (hunk 2 above)
Comment (kilrah) [inline radio/src/translations/en.h.txt]: See line just below :)

File: radio/src/eeprom_rlc.cpp (hunk 1 above)
Comment (kilrah) [inline radio/src/eeprom_rlc.cpp]: Didn't check yet. Figured it would add "a few bytes"...

Comment (kilrah) [inline radio/src/eeprom_rlc.cpp]: It actually saves 2 bytes on my compiler version!

File: radio/src/translations/en.h.txt (hunk 2 above)
Comment (bsongis) [inline radio/src/translations/en.h.txt]: Yes the confirmation is there, but not a contextual menu. I am not sure what is best. Perhaps it's good to keep this as it is, but if we have a new entry to add, we put everything in a menu.

Comment (kilrah) [inline radio/src/translations/en.h.txt]: Oh that, OK. Yes until something else is needed...

**Comment by bsongis** [comment]
Perhaps we add an option in companion, I already introduced something like this for the Bind / Range Check in the special functions.

**Comment by bsongis** [comment]
Ok go, if somebody is not happy we will add a Makefile option

**Pull request merged**